config/util/resource-locker/debug-void.json

```diff
@@ -0,0 +1,14 @@
+{
+    "@context": "https://linkedsoftwaredependencies.org/bundles/npm/@solid/community-server/^2.0.0/components/context.jsonld",
+    "@graph": [
+      {
+        "comment": [
+          "DO NOT USE IN PRODUCTION. ONLY FOR DEVELOPMENT, TESTING, OR DEBUGGING.",
+          "Allows multiple simultaneous read operations and write operations without locks."
+        ],
+        "@id": "urn:solid-server:default:ResourceLocker",
+        "@type": "VoidLocker"
+      }
+    ]
+  }
+  
```

Comment:
@lina1101 for some reason git does not agree with your newline, could you have a look what's going on here? Perhaps a non-unix newline?

Reply:
Done!